schema/va-spec/base/va-core-source.yaml

```diff
@@ -1013,6 +1012,61 @@ $defs:
     - focusAlleleFrequency
     - cohort
 
+  TumorVariantFrequencyStudyResult:
+    inherits: StudyResult
+    maturity: trial use
+    type: object
+    description: >-
+      A Study Result that reports measures related to the frequency of an variant across
+      different tumor types.
+    properties:
+      type:
+        extends: type
+        const: "TumorVariantFrequencyStudyResult"
+        default: "TumorVariantFrequencyStudyResult"
+        description: MUST be "TumorVariantFrequencyStudyResult".
+      sourceDataSet:
+        extends: sourceDataSet
+        description: >-
+          The dataset from which data in the Tumor Variant Frequency Study Result was
+          taken.
+      focusVariant:
+        extends: focus
+        description: The variant for which frequency data is reported in the Study Result
+        oneOf:
+        - $refCurie: vrs:Allele
+        - $refCurie: gks.core:iriReference
+        - $refCurie: cat-vrs:CategoricalVariant
+      affectedTumorSamples:
+        type: integer
+        description: The number of tumor samples that contain the focus variant
+      totalTumorSamples:
```

Comment:
To parallel the CAF Study Result attribute names, we had proposed `totalSampleCount` - was there a reason for the change here?